easybuild/easyconfigs/t/TEtranscripts/TEtranscripts-2.2.0-foss-2020a.eb

```
@@ -34,4 +34,6 @@ sanity_check_paths = {
     'dirs': ['lib/python%(pyshortver)s/site-packages'],
 }
 
+sanity_check_commands = [('TEtranscripts', '--version'), ('TEcount', '--version')]
```

@hajgato No need for tuples, can be just string values with the full command, and we generally use on entry per line:

```suggestion
sanity_check_commands = [
    "TEtranscripts --version",
    "TEcount --version",
]